Formula/code-server.rb

```diff
@@ -28,6 +29,15 @@ class CodeServer < Formula
   def install
     node = Formula["node@14"]
     system "yarn", "--production", "--frozen-lockfile"
+    # @parcel/watcher bundles all binaries for other platforms & architectures
+    # This deletes the non-matching architecture otherwise brew audit will complain.
+    arch = `uname -m`
+    path_to_prebuilds = "vendor/modules/code-oss-dev/node_modules/@parcel/watcher/prebuilds"
+    if arch == "arm64"
+      remove_dir("#{path_to_prebuilds}/darwin-x64")
+    else
+      remove_dir("#{path_to_prebuilds}/darwin-arm64")
+    end
```

Comment:
```suggestion
    prebuilds = testpath/"vendor/modules/code-oss-dev/node_modules/@parcel/watcher/prebuilds"
    (prebuilds/"darwin-x64").unlink if Hardware::CPU.arm?
```
Let's try this; I think we only need to delete this on ARM.

Reply:
woah, you're a wizard!